pkg/generator/generator.go

```diff
@@ -195,6 +204,12 @@ func GetDefaultGeneratorForType(t spansql.Type, cfg data.Config) (data.Generator
 	case spansql.Bool:
 		g, err = data.NewBooleanGenerator(cfg)
 	case spansql.String:
+		// Infer UUID v4 pattern from the column length.
+		if t.Len == uuidV4Length {
+			g, err = data.NewUUIDV4Generator(t.Base, t.Len)
+			break
+		}
```

Comment:
should we also have UUIDV4Generator in bytes ?

Reply:
I thought while `STRING(36)` column is highly likely that it is supposed to have UUID, `BYTES(16)` is less confident that if it's a UUID. If it is actually supposed to have UUID, customers can specify it with configuration. What do you think?

Reply:
customer should track this in user application logic, the schema need not provide hints to the content of the column.

so load generator can create UUID in bytes column if it is specified in the generator.

Reply:
Sounds good! Now this implementation generates UUID value if the customer specifies the UUID type in generator config.